lib/chef/knife/bootstrap.rb

```diff
@@ -366,7 +370,8 @@ def knife_ssh
         ssh.config[:ssh_port] = config[:ssh_port]
         ssh.config[:ssh_gateway] = config[:ssh_gateway]
         ssh.config[:forward_agent] = config[:forward_agent]
-        ssh.config[:identity_file] = config[:identity_file]
+        ssh.config[:identity_file] = config[:identity_file] # DEPRECATED
+        ssh.config[:ssh_identity_file] = config[:ssh_identity_file]
```

Comment:
Perhaps we should *only* set `ssh.config[:ssh_identity_file] = config[:ssh_identity_file] || config[:identity_file]`